<launch>
  <!-- load params -->
  <rosparam
    command="load"
    file="$(find pandora_end_effector_planner)/config/sensor_orientation_planner.yaml"/>

  <node
    name="kinect_orientation_planner"
    pkg="pandora_end_effector_planner"
    type="sensor_orientation_planner"
    respawn="false"
    output="screen"
    args="move_kinect_action"/>

  <node
    name="head_orientation_planner"
    pkg="pandora_end_effector_planner"
    type="sensor_orientation_planner"
    respawn="false"
    output="screen"
    args="move_head_action"/>

</launch>
